api/tests/Serializer/Normalizer/RelatedCollectionLinkNormalizerTest.php

```diff
@@ -265,23 +261,6 @@ public function testNormalizeDoesntReplaceWhenFilterDoesntApplyToMappedProperty(
         $this->shouldNotReplaceChildren($result);
     }
 
-    public function testNormalizeDoesntReplaceWhenStrategyIsNotExact() {
```

Comment:
what happened here?

Reply:
See https://github.com/usu/ecamp3/pull/13
This test has always been flawed, because when filtering by a relation, the strategy of the search filter is always "exact". Before this test used to pass because of other reasons, and when I changed the mocking, it stopped working. So I removed the test.